src/lib.rs

```diff
@@ -210,6 +208,39 @@ impl<'a> Serialize for Bytes<'a> {
     }
 }
 
+struct BytesVisitor;
+
+impl<'de> Visitor<'de> for BytesVisitor {
+    type Value = Bytes<'de>;
+
+    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        formatter.write_str("byte array")
+    }
+
+    #[inline]
+    fn visit_borrowed_bytes<E>(self, v: &'de [u8]) -> Result<Bytes<'de>, E>
+        where E: Error
+    {
+        Ok(Bytes::from(v))
+    }
+
+    #[inline]
+    fn visit_borrowed_str<E>(self, v: &'de str) -> Result<Bytes<'de>, E>
+        where E: Error
+    {
+        Ok(Bytes::from(v.as_bytes()))
```

Comment:
Isn't there a default impl forwarding to visit_ borrowed_ bytes?

Reply:
@oli-obk [`visit_borrowed_str` docs](https://docs.rs/serde/1.0.12/serde/de/trait.Visitor.html#method.visit_borrowed_str) say that it defauts to `visit_str`

Reply:
Ah right. that makes sense